hpke/aead.go

```diff
@@ -0,0 +1,80 @@
+package hpke
+
+import (
+	"crypto/cipher"
+	"errors"
+	"fmt"
+)
+
+type encdecCtx struct {
+	Suite
+	cipher.AEAD
+	baseNonce      []byte
+	seq            []byte
+	exporterSecret []byte
+}
+
+type sealCtx struct{ *encdecCtx }
+type openCtx struct{ *encdecCtx }
+
+func (c *encdecCtx) Export(expCtx []byte, length uint) []byte {
+	maxLength := uint(255 * c.KdfID.Hash().Size())
+	if length > maxLength {
+		panic(fmt.Errorf("size greater than %v", maxLength))
```

Comment:
This behavior needs to be documented. E.g., "Export panics if length is larger than ...."

Reply:
This is documented in the `hpke.Exporter` interface.